jest.config.js

```diff
@@ -10,8 +10,10 @@ module.exports = {
     "**/?(*.)+(spec|test|unit.test).+(ts|tsx|js)"
   ],
   "transform": {
+    "^.+\\.vue$": "vue-jest",
     "^.+\\.tsx?$": "ts-jest"
   },
+  moduleFileExtensions: ['vue', 'js', 'json', 'jsx', 'ts', 'tsx', 'node'],
```

Comment:
No more `jsx` nor `tsx` and what's `node` for?

Reply:
Removed the unused extensions